docs/specs/localization.yml

```diff
@@ -2,92 +2,116 @@
+        docs/child.md:
+  https://github.com/docascode/docfx-test-dependencies.zh-cn#master1:
+    - files:
+        docs/loc.md: |
+          [link to restore-child](~/restore-child/docs/child.md)
 outputs:
+  docs/loc.json:
+  build.log: |
+    ["warning","link-is-dependency","File 'docs/child.md' referenced by link '~/restore-child/docs/child.md' will not be built because it is from a dependency docset","docs/loc.md"]
 ---
 # restore loc repo, all loc files are in one repo
 # [repository + folder mapping]
 commands:
   - restore --locale zh-cn
-  - build
-repo: https://github.com/docascode/docfx-test-dependencies#master
-inputs:
-  docfx.yml: |
-    localization:
-      mapping: RepositoryAndFolder
-restores:
-  git/github.com/docascode/docfx-test-dependencies.localization/18d3d65979413ce8342fd709dbce1c78ab45ffeb-master/dep.md: loc-DEP
-  restore-lock/*restore loc repo, all loc files are in one repo-*-lock.json: |
-    {"git":{"https://github.com/docascode/docfx-test-dependencies.localization#master":"18d3d65979413ce8342fd709dbce1c78ab45ffeb-master"}}
+  - build --locale zh-cn
+repos:
+  https://github.com/docascode/docfx-test-dependencies#master2:
+    - files:
+        docfx.yml: |
+          localization:
+            mapping: RepositoryAndFolder
+  https://github.com/docascode/docfx-test-dependencies.localization#master2:
+    - files:
+        zh-cn/docs/dep.md: loc-DEP
 outputs:
+  docs/dep.json:
 ---
 # loc docset share the source docset's restore map
 commands:
   - restore --locale zh-cn
   - build --locale zh-cn
-repo: https://github.com/docascode/docfx-test-dependencies#loc-restore-map
-inputs:
-  docfx.yml: |
-    github:
-      userCache: github-user.json
-    dependencies:
-      child1: https://github.com/docascode/docfx-test-dependencies.localization#master
-  github-user.json: |
-    {
-      "users": [
-        {
-          "name": "Osmond Jiang",
-          "login": "OsmondJiang",
-          "id": 19990166,
-          "expiry":"2099-01-01T00:00:00.00000Z",
-          "emails": [
-            "xinjiang@microsoft.com"
-          ]
-        },
-        {
-          "name": "Renze Yu",
-          "login": "superyyrrzz",
-          "id": 3831744,
-          "expiry":"2099-01-01T00:00:00.00000Z",
-          "emails": [
-            "renzeyu@microsoft.com",
-            "superyyrrzz@gmail.com"
-          ]
-        },
-        {
-          "name": "Yufei Huang",
-          "login": "yufeih",
-          "id": 511355,
-          "expiry":"2099-01-01T00:00:00.00000Z",
-          "emails": [
-            "yufeih@users.noreply.github.com",
-            "yufeih@live.com"
-          ]
-        }
-      ]
-    }
+repos:
+  https://github.com/docascode/docfx-test-dependencies#share-restore-map:
+    - files:
+        docfx.yml: |
+          github:
+            userCache: github-user.json
+            resolveUsers: true
+          dependencies:
+            child1: https://github.com/docascode/docfx-test-dependencies.localization#master
```

Comment:
also consider making all mocked repos starting with `https://docs.com`, it is hard for me to see whether this dependency is a mocked repo or a real github repo. This helps me better understand our dependencies to https://github.com/docascode/docfx-test-dependencies.

Reply:
some case need to be github.com/xxx to make it a github repo.

---
In reply to: [231411868](https://github.com/dotnet/docfx/pull/3585#discussion_r231411868) [](ancestors = 231411868)